test/sagemaker_tests/pytorch/training/conftest.py

```diff
@@ -435,6 +435,17 @@ def skip_s3plugin_test(request):
             )
 
 
+@pytest.fixture(autouse=True)
```

Comment:
Having both `autouse=True` and `if request.node.get_closest_marker` in this function seems unnecessary.

Reply:
made change